browser/js/app.js

```diff
@@ -8,6 +8,19 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     $urlRouterProvider.otherwise('/');
 });
 
+// Create cookie controller
+
+app.controller('CookieCtrl', ['$scope','$cookieStore', function($scope, $cookieStore) {
+  // Setting a cookie to store the sessionId
+  var sessionId = Math.random().toString(36).slice(2);
```

Comment:
It allows for the inclusion of letters (10 numbers + 26 letters), see: http://code.tutsplus.com/tutorials/understanding-base-36-math--cms-21677.